package/yast2-pam.spec

```diff
@@ -17,27 +17,35 @@
 
 
 Name:           yast2-pam
-Version:        4.2.4
+Summary:	      YaST2 - PAM Agent
```

Comment:
Does not it look like misplaced?

Reply:
What do you mean? The order in the file or the usage of mixed tabs and spaces. Actaully I have a commit  re-aligning all of them, but I wanted to not make too much noise for the review.

Reply:
Anyway, I reformatted it. See https://github.com/yast/yast-pam/pull/20/commits/c113382fe47937b5964b554ce99fda9239a685d9